agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -182,7 +182,12 @@ function install_k8s_agent {
 
     CONFIG_FILE=/tmp/sysdig-agent-configmap.yaml
 
-    echo "* Retreiving the IKS Cluster ID and Cluster Name"
+    if [ $OPENSHIFT -eq 0 ]; then
```

Comment:
Can we simplify things by just saying "Retrieving the Cluster ID and Cluster Name"  ?
Not sure how much value "IKS" adds, the admin probably has other ways of telling what type or orchestrator and cloud provider is being used.

Reply:
I was trying to minimize the changes for the normal non-Openshift case, in case anyone has any scripts or human eyes expecting certain outputs.  But if the consensus is to make the change for all invocations, that's fine with me.